projects/components/src/search-box/search-box.component.ts

```diff
@@ -46,6 +52,9 @@ export class SearchBoxComponent implements OnInit, OnChanges {
   @Input()
   public displayMode: SearchBoxDisplayMode = SearchBoxDisplayMode.Border;
 
+  @Input()
+  public backgroundColor: Color = Color.Transparent;
```

Comment:
can we set backgroundColor in the parent dom element inside code viewer?

Reply:
this property shouldn't be required

Reply:
Removed